proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -96,4 +96,22 @@ message SamplingConfig {
 message Javaagent {
   // set this flag to export certificates configured in JKS to libtraceable for making HTTPS connection to TPA.
   google.protobuf.BoolValue import_jks_certs = 1;
+}
+
+message LogConfig {
+  // logging mode. Allowed values are none, stdout, file.
+  google.protobuf.StringValue mode = 1;
+  // logging level. Allowed values are trace, debug, info, warn, error, critical.
+  google.protobuf.StringValue level = 2;
+  // if logging mode is file, provide this additional configuration
+  LogFileConfig log_file = 3;
```

Comment:
Was wondering if we should rename this to `log_rotation` so that its consistent with TPA and eBPF? https://github.com/Traceableai/traceable-agent/blob/056b2578d7b24a0c643e99b29e2192de8ab9a2cb/proto/config/common/v1/config.proto#L135

Reply:
I feel that rotation is separate to file logging. that way file name is a separate config and log_rotation is separate.

Reply:
like
```
logging:
- level
- mode
- filename
```
and
```
log_rotation:
- max_file_size
- max_file_count
``` 
?

Reply:
does it support oneof?

Reply:
yes, it does

Reply:
Current config looks good to me.